tests/test_logger.py

```diff
@@ -20,10 +20,10 @@ def tearDown(self):
 
     def test_emit(self):
         with mock.patch("iquip.apps.logger._Signaller.signal") as mocked_signal:
-            app = logger.LoggerApp(name="name", parent=QObject())
-            app.setLevel("DEBUG")
+            app = logger.LoggerApp(name="name", logFilePath = "", parent=QObject())
```

Comment:
We do not use whitespaces around `=` keyword arguments.

There are more below, please change all of them.

Reply:
Thank you for review.
I missed it.